core/src/confidence.rs

```diff
@@ -1,14 +1,20 @@
-use crate::result::{Error, Result};
-use crate::service::Service;
+use crate::{
+    result::{Error, Result},
+    rpc::ConfidenceConfig,
```

Comment:
This dependency feels backward.  rpc.rs ought to depend on confidence.rs not vice versa, move ConfidenceConfig in here?

Reply:
I moved it back and forth multiple times ;)